### ISSUE TITLE
Allow arbitrary slot input for Dynamic Impulse Receiver node

### ISSUE THREAD

**Issue by TheBasementNerd** [Neos-Metaverse/NeosPublic#3376]
### Is your tweak request related to a problem? Please describe.

Dynamic Impulse Receivers have to be under a slot to receive an impulse to it, and for my system I'm developing I need to detect an impulse sent to any of the natural body colliders on an avatar

### Describe what would you like tweaked

Add a slot input to Dynamic Impulse Receiver that allows receiving an impulse from any arbitrary slot so that the receiver doesn't have to be parented under the slot

### Describe alternatives you've considered

I am currently having to cycle through all of a user's colliders and parent a receiver system to each one in order to receive hit impulses

**Comment by Beaned2000** [comment]
is having the impulse occur on the root of the avatar and having the transmitter have a name based on the context of the collider (e.g. Colliderhit.Head, Colliderhit.hand) or send it with a value attached to it with the relevant slot not an option?
your use case seems to be pretty doable as things are right now. whether having an input would be an easy implementation or not, i'm not sure but this is definitely something you can solve by changing up the way you are using the dynamic impulses.

**Comment by TheBasementNerd** [comment]
That's true actually, in my use case I could have it just send by grabbing the slot it hits and sending that slot to check if it equals a body node slot for head or chest or something, which would only take a little extra addition on the gun side

**Comment by Beaned2000** [comment]
could i then ask you to check if this works for you and close this issue if this feature is no longer needed?

**Comment by TheBasementNerd** [comment]
Once I get off work I can go ahead and input this method and see how it works with my systems. I still believe the addition to the node would be useful and would appreciate the issue staying open, but for my specific use case I'll check out this idea and report back how well it appears to work
